vrecord

```diff
@@ -2237,11 +2243,6 @@ elif [[ "${RUNTYPE}" = "audiopassthrough" ]] ; then
     exit 0
 fi
 
-# record mode
-if [[ -z "${ID}" ]] ; then
-    _report -d "Starting the vrecord GUI. To bypass the GUI, run like 'vrecord FILENAME'."
-    _edit_mode
-fi
 while [[ -z "${ID}" ]] ; do
```

Comment:
I think this code might also be able to be removed as with the change in behavior to load GUI by default there seems to be no way to trigger it?